content/node-runners/setup/linux.md

```diff
@@ -8,31 +8,66 @@ Hosting a node in a datacenter may ensure better throughput and higher earnings.
 ## Supported operating systems
 - Raspbian 9/10
 - Debian 9/10
-- Ubuntu
+- Ubuntu 18.04/20.04
 
 
 ## Installation
-### Install and start Mysterium Node
-In addition to our Node, this script will check for existing OpenVPN and WireGuard dependencies and install them if missing.
+There are two ways to install a Mysterium node.
+
+On  `Debian`/`Ubuntu`  systems you can use  `Aptitude`  and execute:
+
 
 ```bash
-sudo -E bash -c "$(curl -s https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh)"
+sudo add-apt-repository ppa:mysteriumnetwork/node
+sudo apt-get update
+sudo apt install myst
 ```
 
-### Check service health
-Post installation service check.
+If for some reason it fails, or you can't use Aptitude we also provide an installation script written in `bash` which can be downloaded and executed using this command:
+
+```bash
+NETWORK=testnet2 sudo -E bash -c "$(curl -s https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh)"
```

Comment:
Same here it should be pointed on how to install the latest version, not only testnet2:
```suggestion
sudo -E bash -c "$(curl -s https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh)"
```